### PR TITLE
Install correct Bundler version on Travis CI

### PR DESCRIPTION
**Pull request by enricostano**
Related to https://github.com/coopdevs/timeoverflow-provisioning/pull/69 but not dependent.

### CODE REVIEW

**Comment by enricostano** [comment]
@markets @Morantron do you have any idea why the sign in features spec is failing in `develop`? Could be related to Chromedriver version?

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ env:
   global:
     - DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
     - CC_TEST_REPORTER_ID=025bc15a0fa9afa52d86ee24fea845cf1d363f48a466bcf2cef8ab80c29acb28
+before_install:
+  - gem install bundler:1.17.0
```
Comment (Morantron) [inline .travis.yml]: from `Gemfile.lock` :eyes: 

```
BUNDLED WITH
   1.16.2
```

Comment (enricostano) [reply .travis.yml]: https://github.com/coopdevs/timeoverflow/blob/develop/Gemfile.lock#L441

**Comment by sseerrggii** [comment]
giphy (3)

**Comment by markets** [comment]
@enricostano Solution proposal in #507

**Pull request merged**